release/pkg/manager/calico/manager.go

```diff
@@ -213,10 +220,10 @@ type CalicoManager struct {
 	githubToken string
 }
 
-func releaseImages(version, operatorVersion string) []string {
-	imgList := []string{fmt.Sprintf("quay.io/tigera/operator:%s", operatorVersion)}
-	for _, img := range append(images, windowsImages...) {
-		imgList = append(imgList, fmt.Sprintf("%s:%s", img, version))
+func releaseImages(images []string, version, registry, operatorImage, operatorVersion, operatorRegistry string) []string {
+	imgList := []string{fmt.Sprintf("%s/%s:%s", operatorRegistry, operatorImage, operatorVersion)}
```

Comment:
This is a change worth talking about - the operator isn't actually produced as part of a Calico release, so I think this is technically wrong? What's the motivation for including it here?

Reply:
operator was previously included with the registry and image hardcoded to `quay.io` and `tigera/operator` respectively which is not always correct esp if the user updates the values when building operator.